### ISSUE TITLE
Wallet Directory

### ISSUE THREAD

**Issue by flangelier** [csgames/cscoins#34]
In the Wallet Manager, It would be awesome if I could link some label or name (Like the University name) to the wallet ID. So I could see the list of University and their transaction instead of their wallet ID

**Comment by Cyberunner23** [comment]
Is this still rewardable?

**Comment by flangelier** [comment]
Yes, you'll always have a special place in my heart! (But the CA is down so you won't have any CS Coins)
